clang/unittests/Support/TimeProfilerTest.cpp

```diff
@@ -81,7 +81,6 @@ std::string GetMetadata(json::Object *Event) {
   if (json::Object *Args = Event->getObject("args")) {
     if (auto Detail = Args->getString("detail"))
       OS << Detail;
-    // Use only filename to not include os-specific path separators.
```

Comment:
Is this unintentional? Probably warrants a revert.

Reply:
Reverted.